pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ApexTreeBuilder.kt

```diff
@@ -17,15 +17,30 @@ import com.google.summit.ast.TypeRef
 import com.google.summit.ast.declaration.ClassDeclaration
 import com.google.summit.ast.declaration.EnumDeclaration
 import com.google.summit.ast.declaration.InterfaceDeclaration
+import com.google.summit.ast.declaration.MethodDeclaration
+import com.google.summit.ast.declaration.PropertyDeclaration
 import com.google.summit.ast.declaration.TriggerDeclaration
 import com.google.summit.ast.declaration.TypeDeclaration
 import com.google.summit.ast.modifier.KeywordModifier
+import com.google.summit.ast.modifier.KeywordModifier.Keyword
 import com.google.summit.ast.modifier.Modifier
+import com.google.summit.ast.statement.CompoundStatement
 
 @Deprecated("internal")
 @InternalApi
 @Suppress("DEPRECATION")
 class ApexTreeBuilder(val sourceCode: String, val parserOptions: ApexParserOptions) {
+    companion object {
+        /** Name used for [MethodDeclaration]s that are constructors. */
+        private const val JORJE_CONSTRUCTOR_ID = "<init>"
```

Comment:
I'm not clear as to why you are using "JORJE" in the name of these constants.

Reply:
Many rules expect the names of certain method declarations (namely constructors, static init blocks, and property accessors) to match those produced by Jorje, so I am replacing Summit names with Jorje/PMD names during tree building. I can make this more clear in the documentation. Maybe I'll also replace `JORJE_` with `PMD_` as that represents the purpose better.

Reply:
I suggest moving the string constant to the PMD ASTMethod, since either its internal methods or its callers also need to know what the magic string is.

Reply:
Currently, rules have these strings hard-coded, but I agree that this would be a better location for them so that rules could be refactored in the future.